packages/axelar-std/src/interfaces/mod.rs

```diff
@@ -8,6 +8,12 @@ pub use operatable::*;
 pub use ownable::*;
 pub use upgradable::*;
 
+/// Marker trait for interfaces that should not be implemented by using .
```

Comment:
```suggestion
/// Marker trait for interfaces that should not be implemented by using `contractimpl`.
```
missing word i think